data_model/src/account.rs

```diff
@@ -461,7 +454,9 @@ impl FromStr for Id {
 
     fn from_str(string: &str) -> Result<Self, Self::Err> {
         if string.is_empty() {
```

Comment:
I think this check is unnecessary because the next check for the correct format will always fail with an empty string, but it's useful in terms of error readability. 
Should we keep it? @ales-tsurko what do you think?

Reply:
I think this check is unnecessary because the next check for the correct format will always fail with an empty string, but it's useful in terms of error readability.
Should we keep it? @ales-tsurko what do you think?

It has different `reason` description. The first is more specific and related to particular case (i.e. empty string).

Reply:
ok, resolve then